pkg/reducer/interface.go

```diff
@@ -23,11 +23,25 @@ type IntervalWindow interface {
 	EndTime() time.Time
 }
 
+// CreateReducer is the interface which is used to create a Reducer.
+type CreateReducer interface {
```

Comment:
`CreateReducer` is a verb, is it good for interface name? `ReducerFactory`?

Reply:
Let's go with `ReducerCreater`?